pwa/src/pages/cronjobs/[cronjobId]/CronjobDetailPage.tsx

```diff
@@ -0,0 +1,14 @@
+import * as React from "react";
```

Comment:
(This comment is regarding more than just this file. It's about the structure from forms and its detail pages and templates).

As discussed in our call today, we want to remove the need to _duplicate_ forms. Right now, we have the same form in `EditCronjobFormTemplate` and `CreateCronjobFormTemplate`. We want to have only _one_ form file per resource.

We expect something like this:

```
CronjobDetailPage
  => CronjobDetailTemplate => CronjobFormTemplate
  => CronjobCreateTemplate => CronjobFormTemplate
```

Where the `CronjobDetailTemplate` is responsible for the `Delete` and `AddToDashboard` buttons.

And the `CronjobFormTemplate` is responsible for the `onSubmit` function, this template should have an optional `cronjob` prop, where the cronjob _can_ be passed. If it is passed, it should be used to (1) `setValue` of the entire form and (2) it's `id` should be passed to the `createOrUpdate` function.

Reply:
The layout would become (something like):

![image](https://user-images.githubusercontent.com/66552932/207844146-52b06ec0-7b75-42a1-88e0-35ad5d840d6a.png)